src/Cli/dotnet/Telemetry/TelemetryFilter.cs

```diff
@@ -25,9 +25,19 @@ public IEnumerable<ApplicationInsightsEntryFormat> Filter(object objectToFilter)
         {
             var result = new List<ApplicationInsightsEntryFormat>();
 
+            if (objectToFilter is ParseResult topLevelParseResult)
+            {
+                var topLevelCommandName = topLevelParseResult.RootCommandResult.Children.FirstOrDefault()?.Symbol.Name;
```

Comment:
This could be an option or argument but it looks like we're expecting a command.